recipes/scala/build.sh

```diff
@@ -0,0 +1 @@
+mvn clean install
```

Comment:
It's not clear that we actually need to call `install` here since we don't need the artifacts installed on the local system (which goes to maven's common storage location).  The `COPY` command in the Dockerfile is copying the assets from the `target` folder.  We should use minimal effort strategy and call something like, `mvn clean build` or equivalent as a potential alternative.

Reply:
Good catch, we would need the mvn clean install to get the fat jar locally (and use this to create an engine) and actually do not need to copy the assets since this is an external workflow. Will confirm if this is the case going forward.